Makefile-libs.Inc

```diff
@@ -71,7 +76,12 @@ $(BINDIR_POSTRS_SETUP_LIBS): $(PROJ_DIR)$(POSTRS_SETUP_ZIP)
 $(PROJ_DIR)$(POSTRS_SETUP_ZIP):
 	curl -L $(POSTRS_SETUP_URL_ZIP) -o $(PROJ_DIR)$(POSTRS_SETUP_ZIP)
 
+$(BIN_DIR)$(POSTRS_PROFILER_BIN):
+	curl -L $(POSTRS_PROFILER_URL) -o $(PROJ_DIR)$(POSTRS_PROFILER_ZIP)
+	unzip -o -j $(PROJ_DIR)$(POSTRS_PROFILER_ZIP) -d $(BIN_DIR)
```

Comment:
I'd suggest renaming it to something like `post_proving_profiler.exe` because just `profiler.exe` is ambiguous (what does it profile?)